R/archive.R

```diff
@@ -584,23 +584,24 @@ epi_archive =
 #'   details. 
 #' @importFrom data.table key
 #' @importFrom rlang !! !!! enquo quo_is_missing enquos is_quosure sym syms
-          slide = function(f, ..., n, group_by, ref_time_values, 
+          slide = function(f, ..., max_version_gap, group_by,
+                           ref_versions, 
                            time_step, new_col_name = "slide_value",
                            as_list_col = FALSE, names_sep = "_",
                            all_rows = FALSE) { 
             # If missing, then set ref time values to be everything; else make
             # sure we intersect with observed time values 
-            if (missing(ref_time_values)) {
-              ref_time_values = unique(self$DT$time_value)
+            if (missing(ref_versions)) {
+              ref_versions = unique(self$DT$version)
```

Comment:
Based on discussion with Ryan, we shouldn't rename `ref_time_values` (yet, at least).